library/a-star/metadata/9th-floor-hall.json

```diff
@@ -0,0 +1 @@
-925-2":{"0":{"x":12,"y":23},"1":{"x":12,"y":24},"2":{"x":13,"y":23},"3":{"x":13,"y":24},"4":{"x":14,"y":23},"5":{"x":14,"y":24}},"H-931":{"0":{"x":19,"y":21},"1":{"x":20,"y":21}},"H-927-3":{"0":{"x":15,"y":23},"1":{"x":15,"y":24}},"H-961-9":{"0":{"x":17,"y":0}},"H-961-10":{"0":{"x":17,"y":2},"1":{"x":17,"y":3}},"H-961-8":{"0":{"x":15,"y":2},"1":{"x":15,"y":3},"2":{"x":17,"y":4},"3":{"x":18,"y":4}},"H-925-1":{"0":{"x":13,"y":21},"1":{"x":13,"y":22}},"H-961-7":{"0":{"x":15,"y":0}},"H-961-12":{"0":{"x":18,"y":2},"1":{"x":18,"y":3}},"H-961-6":{"0":{"x":14,"y":2},"1":{"x":14,"y":3}},"H-961-11":{"0":{"x":18,"y":0}},"H-933":{"0":{"x":21,"y":18},"1":{"x":21,"y":19},"2":{"x":22,"y":18},"3":{"x":22,"y":19},"4":{"x":23,"y":17},"5":{"x":23,"y":18},"6":{"x":23,"y":19}},"escalator":{"0":{"x":10,"y":11},"1":{"x":11,"y":11},"2":{"x":12,"y":11},"3":{"x":13,"y":11}},"H-927-4":{"0":{"x":15,"y":21},"1":{"x":15,"y":22},"2":{"x":16,"y":21}},"H-981":{"0":{"x":18,"y":8}},"H-980":{"0":{"x":6,"y":13},"1":{"x":6,"y":14},"2":{"x":7,"y":13},"3":{"x":7,"y":14},"4":{"x":8,"y":13},"5":{"x":8,"y":14},"6":{"x":9,"y":13},"7":{"x":9,"y":14},"8":{"x":10,"y":13},"9":{"x":10,"y":14},"10":{"x":11,"y":13},"11":{"x":11,"y":14},"12":{"x":12,"y":13},"13":{"x":12,"y":14},"14":{"x":13,"y":13},"15":{"x":13,"y":14},"16":{"x":13,"y":15},"17":{"x":13,"y":16},"18":{"x":13,"y":17},"19":{"x":13,"y":18},"20":{"x":14,"y":13},"21":{"x":14,"y":14},"22":{"x":14,"y":15},"23":{"x":14,"y":16},"24":{"x":14,"y":17},"25":{"x":14,"y":18}},"H-907":{"0":{"x":1,"y":8},"1":{"x":1,"y":9},"2":{"x":1,"y":10},"3":{"x":1,"y":11},"4":{"x":2,"y":8},"5":{"x":2,"y":9},"6":{"x":2,"y":10},"7":{"x":2,"y":11},"8":{"x":3,"y":8},"9":{"x":3,"y":9},"10":{"x":3,"y":10},"11":{"x":3,"y":11},"12":{"x":4,"y":8},"13":{"x":4,"y":9},"14":{"x":4,"y":10},"15":{"x":4,"y":11}},"H-906":{"0":{"x":6,"y":11}},"H-909":{"0":{"x":1,"y":12},"1":{"x":1,"y":13},"2":{"x":1,"y":14},"3":{"x":2,"y":12},"4":{"x":2,"y":13},"5":{"x":2,"y":14},"6":{"x":3,"y":12},"7":{"x":3,"y":13},"8":{"x":3,"y":14},"9":{"x":4,"y":12},"10":{"x":4,"y":13},"11":{"x":4,"y":14}},"H-908":{"0":{"x":7,"y":11}},"H-986":{"0":{"x":8,"y":18},"1":{"x":8,"y":19}},"H-929-25":{"0":{"x":22,"y":12},"1":{"x":22,"y":13},"2":{"x":22,"y":14},"3":{"x":23,"y":12},"4":{"x":23,"y":13},"5":{"x":23,"y":14}},"H-941":{"0":{"x":21,"y":10},"1":{"x":22,"y":9},"2":{"x":22,"y":10},"3":{"x":22,"y":11},"4":{"x":23,"y":9},"5":{"x":23,"y":10},"6":{"x":23,"y":11}},"H-903":{"0":{"x":1,"y":4},"1":{"x":1,"y":5},"2":{"x":1,"y":6},"3":{"x":1,"y":7},"4":{"x":2,"y":4},"5":{"x":2,"y":5},"6":{"x":2,"y":6},"7":{"x":2,"y":7},"8":{"x":3,"y":4},"9":{"x":3,"y":5},"10":{"x":3,"y":6},"11":{"x":3,"y":7},"12":{"x":4,"y":4},"13":{"x":4,"y":5},"14":{"x":4,"y":6},"15":{"x":4,"y":7}},"H-902":{"0":{"x":6,"y":7},"1":{"x":6,"y":8},"2":{"x":6,"y":9},"3":{"x":7,"y":7},"4":{"x":7,"y":8},"5":{"x":7,"y":9},"6":{"x":7,"y":10}},"H-932-2":{"0":{"x":19,"y":18}},"H-945":{"0":{"x":19,"y":5},"1":{"x":19,"y":6},"2":{"x":19,"y":7},"3":{"x":19,"y":8}}}}
```

Comment:
this is an old version and shouldn't be includes. Switch 9th-floor-test to this name to match naming conventions